### PR TITLE
chore: small adaptions to docs + docstrings

### PR DESCRIPTION
**Pull request by jnussbaum**

### CODE REVIEW

File: docs/excel2xml-module.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ here](./file-formats/xml-data-file.md#dsp-base-resources-and-base-properties-to-
 #### Resource ID
 
 Special care is needed when the ID of a resource is created. Every resource must have an ID that is unique in the file,
-and it must meet the constraints of xsd:ID. You can simply achieve this if you use the method `make_xsd_id_compatible()`.
+and it must meet the constraints of xsd:ID. You can simply achieve this if you use the function `make_xsd_id_compatible()`.
```
Comment (Nora-Olivia-Ammann) [inline docs/excel2xml-module.md:199]: You should mention here that it also adds a UUID based on the timestamp which cannot be reconstructed later.

File: docs/excel2xml-module.md [hunk 2]
```diff
@@ -266,7 +266,7 @@ For `make_boolean_prop(cell)`, the following formats are supported:
 - false: False, "false", "False", "0", 0, "no", "No"
 
 N/A-like values will raise an Error. 
-So if your cell is empty, this method will not count it as false, 
+So if your cell is empty, this function will not count it as false,
```
Comment (Nora-Olivia-Ammann) [inline docs/excel2xml-module.md:269]: Why do you change it from method to function here and above the reverse? According to this: https://www.shiksha.com/online-courses/articles/difference-between-methods-and-functions-in-python/ currently we do not at all offer methods in `excel2xml-lib`.

File: docs/file-formats/xml-data-file.md [hunk 3]
```diff
@@ -986,15 +986,15 @@ shortcut, so that the XML file can be validated more precisely.
...
-| `hasTitle`         | 0-1         | text-prop     | utf8     |
-| `hasDescription`   | 0-n         | text-prop     | xml      |
-| `hasComment`       | 0-n         | text-prop     | xml      |
-| `hasKeyword`       | 0-n         | text-prop     | utf8     |
-| `relatesTo`        | 0-n         | resptr-prop   |          |
+| prop name          | cardinality | prop type     | text type   |
+| ------------------ | ----------- | ------------- | ----------- |
+| `isSegmentOf`      | 1           | resptr-prop   |             |
+| `hasSegmentBounds` | 1           | interval-prop |             |
+| `hasTitle`         | 0-1         | text-prop     | unformatted |
```
Comment (Nora-Olivia-Ammann) [inline docs/file-formats/xml-data-file.md:993]: Perhaps write the encoding in the brackets. Till we made that distinction without them having to enter the encoding, I would mention it in the documentation.

**Review by Nora-Olivia-Ammann — CHANGES_REQUESTED** [review]

File: docs/excel2xml-module.md (hunk 2 above)
Comment (jnussbaum) [reply docs/excel2xml-module.md:269]: The change I did was from method -> function, but not the reverse. Function is the correct term.

There is no change in this PR that turns function -> method.

**Review by Nora-Olivia-Ammann — APPROVED** [review]

File: docs/excel2xml-module.md (hunk 1 above)
Comment (jnussbaum) [reply docs/excel2xml-module.md:199]: May I delegate this work to you, in https://linear.app/dasch/issue/DEV-4026/featexcel2xml-new-function-make-xsd-id-compatible-without-uuids?

File: docs/file-formats/xml-data-file.md (hunk 3 above)
Comment (jnussbaum) [reply docs/file-formats/xml-data-file.md:993]: Here, they don't have to enter the encoding, so it's better to omit the encoding

**Pull request merged**